### DISCUSSION TITLE
Personachat in ParlAI Dialog Format

### DISCUSSION THREAD

**Discussion by cahya-wirawan** [facebookresearch/ParlAI#4359] [Q&A] [answered]
Hi, I want to finetune my non english gpt-2 model with personachat dataset, therefore I have to translate it first to my language. I just don't know how to save this dataset as ParlAI Dialog text Format. Currently I know only about "text", "labels" and "episode_done" in the dialog format, how about other entries like text candidates and the persona, how can I store it in ParlAI Dialog Format? 
Or would it be better to store the translations in the same format like the text file in "ParlAI/data/ConvAI2" directory and set the "datapath" apropriately?
Thanks

**Comment by klshuster** [comment, marked as answer]
You can include those the same way as the other fields; everything will be read into a message from the line. You can separate candidates using the `|` character

**Reply by cahya-wirawan** [reply]
Ok, thanks
